XVim/XVimExCommand.m

```diff
@@ -1045,7 +1045,7 @@ - (void)ncounterpart:(XVimExArg*)args inWindow:(XVimWindow*)window{
     // We must make forcus back to editor first then invoke the command.
     // This is because I do not know how to move focus on newly visible text editor by invoking this command.
     // Without this focus manipulation the focus after the command does not goes to the text editor
-    //[window setForcusBackToSourceView];
+    [window setForcusBackToSourceView];
```

Comment:
Does this mean that you comment in the commented out line?
I don't remember why I commented out the line but there should be a reason...